docs/index.rst

```diff
@@ -37,6 +37,7 @@ Learn more from :doc:`the documentation <manual/introduction>`.
 
    manual/introduction
    manual/installation
+   manual/upgrading-to-phpspec-3
```

Comment:
this should come after the getting starting guide IMO. It should probably be the last chapter